examples/opengl_underlay/scene.60

```diff
@@ -0,0 +1,34 @@
+// Copyright © SixtyFPS GmbH <info@sixtyfps.io>
+// SPDX-License-Identifier: (GPL-3.0-only OR LicenseRef-SixtyFPS-commercial)
+
+import { ScrollView, Button, CheckBox, SpinBox, Slider, GroupBox, LineEdit, StandardListView,
+    ComboBox, HorizontalBox, VerticalBox, GridBox, TabWidget, TextEdit, AboutSixtyFPS } from "sixtyfps_widgets.60";
+
+App := Window {
+    preferred-width: 500px;
+    preferred-height: 600px;
+    title: "SixtyFPS Gallery";
```

Comment:
```suggestion
```

Not knowing what title to use, i'd leave it the default (so it will be "SixtyFPS Window").
But we could also give a better title

Reply:
You're right, that should be removed.